docs/modules/ROOT/pages/api/account.adoc

```diff
@@ -726,6 +727,7 @@ include::../utils/_class_hashes.adoc[]
 .AccountComponent
 
 * xref:#AccountComponent-Embeddable-Mixin-Impl[`++AccountMixinImpl++`]
+* xref:#SRC9Component-Embeddable-Impls-OutsideExecutionV2Impl[`++OutsideExecutionV2Impl++`]
```

Comment:
```suggestion

.SRC9Component

* xref:#SRC9Component-Embeddable-Impls-OutsideExecutionV2Impl[`++OutsideExecutionV2Impl++`]
```
We should have SRC9Component on the right side of `OutsideExecutionV2Impl`, no?

Reply:
Same with eth account if we agree

Reply:
Good catch!